client/my-sites/checkout/checkout-thank-you/index.jsx

```diff
@@ -532,6 +543,27 @@ export class CheckoutThankYou extends Component {
 		);
 	}
 
+	getProfessionalEmailFromPurchase( wasDomainMappingOrRegistration, purchases ) {
+		if ( ! wasDomainMappingOrRegistration ) {
+			return null;
+		}
+
+		if ( purchases.filter( isTitanMail ).length > 0 ) {
+			return purchases.filter( isTitanMail )[ 0 ];
+		}
```

Comment:
It might be cleaner to capture the `filter` output in a variable.
```suggestion
		const titanMailPurchases = purchases.filter( isTitanMail );
		if ( titanMailPurchases.length > 0 ) {
			return titanMailPurchases[ 0 ];
		}
```